ts_benchmark/evaluation/strategy/fixed_forecast.py

```diff
@@ -73,8 +74,10 @@ def _execute(
         inference_data = pd.DataFrame(
             predicted, columns=test_data.columns, index=test_data.index
         )
-        actual_data_encoded = self._encode_data(test_data)
-        inference_data_encoded = self._encode_data(inference_data)
+
+        save_true_pred = self._get_scalar_config_value("save_true_pred", series_name)
```

Comment:
is it necessary to add 'save_true_pred' to the REQUIRED_CONFIGS variable?

Reply:
it has been added